python/ray/serve/scripts.py

```diff
@@ -454,6 +466,16 @@ def run(
             visualizer = GraphVisualizer()
             visualizer.visualize_with_gradio(handle)
         else:
+            if reload:
+                for changes in watchfiles.watch(
+                    working_dir,
```

Comment:
what happens if `working_dir` is not provided? for local development, maybe let's default to `"."` and/or pick up the `--app-dir` if it's specified?

Reply:
I have added to defaulting to `./` if the working directory is not specified.